data/src/main/java/org/cryptomator/data/cloud/crypto/CryptoImplDecorator.kt

```
@@ -332,16 +426,120 @@ abstract class CryptoImplDecorator(
+				}
+				closeQuietly(thumbnailReader)
+
+				cryptoFile.thumbnail = diskCache[cacheKey]
+			} catch (e: Exception) {
+				Timber.e(e, "Bitmap generation crashed")
+			}
+		}
+	}
+
+	protected fun generateCacheKey(cryptoFile: CryptoFile): String {
+		return String.format("%s-%d", cryptoFile.cloudFile.cloud?.id() ?: "common", cryptoFile.path.hashCode())
```

detekt

[warning] String.format("%s-%d", cryptoFile.cloudFile.cloud?.id() ?: "common", cryptoFile.path.hashCode()) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)